dbms/src/Common/config_version.h.in

```diff
@@ -13,10 +13,22 @@
     #cmakedefine VERSION_REVISION @VERSION_REVISION@
 #endif
 
-#cmakedefine VERSION_STRING "@VERSION_STRING@"
-#cmakedefine VERSION_FULL "@VERSION_FULL@"
-#cmakedefine VERSION_DESCRIBE "@VERSION_DESCRIBE@"
-#cmakedefine VERSION_GITHASH "@VERSION_GITHASH@"
+#cmakedefine TIFLASH_VERSION_MAJOR @TIFLASH_VERSION_MAJOR@
+#ifndef TIFLASH_VERSION_MAJOR
+#define TIFLASH_VERSION_MAJOR 0
+#endif
```

Comment:
why we need these lines?

Reply:
When TIFLASH_VERSION_MAJOR = 0, cmakedefine output `/* #undef VAR */` rather than `define VAR ...`

Reply:
use those lines to make sure `TIFLASH_VERSION_MAJOR` will always be define.